tests/cypress/integration/classic-create-post.test.js

```diff
@@ -0,0 +1,87 @@
+const { randomName } = require('../support/functions');
+
+describe('Command: classicCreatePost', () => {
+  before(() => {
+    cy.login();
+    cy.activatePlugin('classic-editor');
```

Comment:
I think we should add `cy.activatePlugin('classic-editor');` in **after()** function, as most of our command in this lib uses block editor, what do you think?

Reply:
@iamdharmesh thank you for the question. [Clean up state before tests run, not after](https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks) best practice is applied across all tests of this utility library. We are disabling Classic Editor before all tests which require Block Editor.